here-naksha-lib-view/src/main/java/com/here/naksha/lib/view/IView.java

```diff
@@ -20,4 +20,7 @@
 
 import com.here.naksha.lib.core.storage.IStorage;
 
-public interface IView extends IStorage {}
+public interface IView extends IStorage {
+
+  void setViewLayerCollection(ViewLayerCollection viewLayerCollection);
```

Comment:
I remember we discussed making View immutable - I also remember we agreed on this setter to be here in the first iteration. Would you mind adding a task into "nice-to-haves backlog" that would address:
- getting rid of this setter
- such instantiation flow (constructor + PluginCache + call-site) that guarantees that each View instance is ready to be used right after being created (aka: we pass `viewLayerCollection` as parameter and not allow any setter on `View`)

Reply:
MCPODS-7046